.env.test

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://localhost/laa_court_data_adaptor_test
```

Comment:
we probably don't want this added here as this might interfere with the test setup